src/tests/elua/meson.build

```diff
@@ -6,12 +6,6 @@ elua_suite_src = [
    'elua_lib.c'
 ]
 
-if not sys_windows
-  elua_bindings_dir = join_paths(meson.source_root(), 'src', 'bindings', 'luajit')
-else
-  elua_bindings_dir = join_paths(meson.source_root(), 'src', 'bindings', 'lua')
-endif
-
```

Comment:
Was this even doing something? I mean, there is a `elua_bindings_dir = join_paths(meson.source_root(), 'src', 'bindings', 'lua')` right after this.

Reply:
I asked myself the same thing actually. I think it was a previous conflict resolution that let both exist at the same time.

Reply:
Shouldn't we remove the L15 instead?

Reply:
No, because that if doesn't even make sense. "Lua" is a language and then there's a binding to it. "luajit" is just a lua interpreter, a runtime platform. `src/bindings/luajit` doesn't even exist.

Reply:
oh, okay